azure-devops/azext_devops/dev/boards/work_item.py

```diff
@@ -242,10 +246,15 @@ def query_work_items(wiql=None, id=None, path=None, organization=None, project=N
         id = query.id
     if id is not None:
         query_result = client.query_by_id(id=id)
-    else:
+    if wiql is not None:
         wiql_object = Wiql()
         wiql_object.query = wiql
         query_result = client.query_by_wiql(wiql=wiql_object)
+    if wiql_file_path is not None:
```

Comment:
:Nit 

We can combine wiql and wiql file path input similar to how path and id are combined. 
```
if wiql is None and wiql_file_path is not None:
    with open(wiql_file_path, 'r') as file:
        wiql = file.read().replace('\n', '')

Then this remains as is.. 
if wiql is not None:
    wiql_object = Wiql()
    wiql_object.query = wiql
    query_result = client.query_by_wiql(wiql=wiql_object)
```